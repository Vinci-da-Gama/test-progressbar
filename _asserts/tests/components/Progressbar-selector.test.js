import React from 'react';
import { shallow } from 'enzyme';

import ProgressbarSelector from '../../src/components/Progressbar-selector';
import barsArr from '../fixtures/barsArr-fixtures';

describe('In components/Progressbar-selector.js Test Selector: ', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<ProgressbarSelector hasSelectedPb={ barsArr[0].pbId } bars={ barsArr } />);
    });

    it('should render Progressbar-selector correctly.', () => {
        expect(wrapper).toMatchSnapshot();
    });

});
