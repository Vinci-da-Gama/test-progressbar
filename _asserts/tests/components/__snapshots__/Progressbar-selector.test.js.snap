// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`In components/Progressbar-selector.js Test Selector:  should render Progressbar-selector correctly. 1`] = `
<Input
  bsSize="lg"
  className="rounded-0 normal-bdcolor-boxshadow custom-select height-3rem appear-btn"
  id="progressbAs-select0"
  name="progressbar-select"
  onChange={[Function]}
  type="select"
>
  <option
    disabled={true}
    value="default"
  >
    Please Select
  </option>
  <PbOpts
    key="0"
    pbId="pb1"
    val={10}
  />
  <PbOpts
    key="1"
    pbId="pb2"
    val={81}
  />
  <PbOpts
    key="2"
    pbId="pb3"
    val={42}
  />
</Input>
`;
